test/ruby/test_iseq.rb

```diff
@@ -798,4 +798,15 @@ def test_ibf_bignum
     result = RubyVM::InstructionSequence.load_from_binary(iseq.to_binary).eval
     assert_equal expected, result, proc {sprintf("expected: %x, result: %x", expected, result)}
   end
+
+  def test_compile_prism_with_file
+    Tempfile.create(%w"test_compile_file .rb") do |f|
```

Comment:
I'm not sure what's going on with the tempfile/windows, but maybe `File.open(__FILE__)` would work instead?